carma/launch/environment.launch.py

```diff
@@ -69,6 +69,37 @@ def generate_launch_description():
         get_package_share_directory('motion_computation'), 'config/parameters.yaml')
 
     env_log_levels = EnvironmentVariable('CARMA_ROS_LOGGING_CONFIG', default_value='{ "default_level" : "WARN" }')
+    
+    carma_wm_ctrl_ros2_param_file = os.path.join(
+        get_package_share_directory('carma_wm_ctrl_ros2'), 'config/parameters.yaml')
+
+    carma_wm_ctrl_container = ComposableNodeContainer(
```

Comment:
This doesn't need it's own container. Instead, just the ComposableNode should be added to external_objects_container to take advantage of intra-process comms.